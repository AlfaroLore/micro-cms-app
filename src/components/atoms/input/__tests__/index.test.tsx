import { render } from '@testing-library/react';
import Input from '..';

describe('Input', () => {
  it('Renders Input', () => {
    const component = render(<Input />);

    expect(component).toBeTruthy();
    expect(component.asFragment()).toMatchSnapshot();
  });
});
