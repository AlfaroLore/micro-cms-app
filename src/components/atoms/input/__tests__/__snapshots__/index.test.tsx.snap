// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Input > Renders Input 1`] = `
<DocumentFragment>
  <input
    class="sc-bgqQcB eoSoDA"
  />
</DocumentFragment>
`;
